src/pal/pal_posix.h

```diff
@@ -19,6 +19,11 @@
 #  include <unistd.h>
 #endif
 
+// default value for builds not using CMake
+#ifndef SNMALLOC_PLATFORM_HAS_GETENTROPY
+#  define SNMALLOC_PLATFORM_HAS_GETENTROPY 0
+#endif
```

Comment:
It might be nice to add a `#warning` here, something like:
```suggestion
#ifndef SNMALLOC_PLATFORM_HAS_GETENTROPY
#  define SNMALLOC_PLATFORM_HAS_GETENTROPY 0
#  warning snmalloc not using entropy source. To silence this warning please define SNMALLOC_PLATFORM_HAS_GETENTROPY to 0, or to Entropy if your platform implements getentropy().
#endif
```